packages/e2e/test/blockfrost-providers/networkInfo.test.ts

```diff
@@ -24,3 +27,54 @@ describe.skip('Web Socket', () => {
     expect(toSerializableObject(response)).toEqual(toSerializableObject(legacyResponse));
   });
 });
+
+describe('BlockfrostNetworkInfoProvider', () => {
+  let factory: Promise<NetworkInfoProvider>;
+
+  beforeAll(() => {
+    factory = networkInfoProviderFactory.create(
+      'blockfrost',
+      {
+        baseUrl: 'https://cardano-preprod.blockfrost.io',
+        projectId: env.BLOCKFROST_API_KEY
```

Comment:
We should either make this a 'preprod test suite' or run against the local blockfrost (`blockfrost-ryo`)? I think running against local is better, but probably a little bit more work. Any reason besides practicality to run these tests against the hosted blockfrost?

```suggestion
        projectId: env.BLOCKFROST_API_KEY_PREPROD
```

Reply:
> Any reason besides practicality to run these tests against the hosted blockfrost?

Yeah I was hoping we could just get coverage using wallet e2e tests against the BYO, but didn't want to burn time on it since we can move forward and come back to this. If you see this as an easy win, go for it

Reply:
Decided to remove provider tests and rely on wallet tests only. There was significant overlap. See a97c296e5628c9a50180e82d3ee10e9d17b519cc